nixos/modules/services/security/authelia.nix

```diff
@@ -74,6 +74,30 @@ let
                 '';
               };
 
+              notifierSmtpPasswordFile = mkOption {
+                type = types.nullOr types.path;
```

Comment:
`types.path` is usually not what you want: this accepts path literals and when somebody uses them wrongly (i.e. `"${toString pathLiteral}"` vs `"${pathLiteral}"`) you end up silently copying things into your store.

`pathWith` is what you'll probably want.

Reply:
Good to know! The existing options use `types.path` as well -- do you think it's appropriate for this PR to address those options, too?